sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-additional-transportation-costs.e2e-spec.ts

```diff
@@ -0,0 +1,370 @@
+      "and studentDataAdditionalTransportPlacement is true and offeringDelivered is onsite",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 100;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 20;
+      assessmentConsolidatedData.studentDataAdditionalTransportCost = 80;
+      assessmentConsolidatedData.studentDataAdditionalTransportPlacement = true;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Onsite;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataAdditionalTransportationMax = min((94-25), 80) * min(30, 20) * 0.34 = 469.2
+      // calculatedDataTotalTransportationAllowance = (30 - 20) * 13 + 469.2 = 599.2
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportationMax,
+      ).toBe(469.2);
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(599.2);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm less than 280 " +
+      "and studentDataAdditionalTransportPlacement is true and offeringDelivered is online",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 100;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.studentDataAdditionalTransportWeeks = 20;
+      assessmentConsolidatedData.studentDataAdditionalTransportCost = 80;
+      assessmentConsolidatedData.studentDataAdditionalTransportPlacement = true;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Online;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataAdditionalTransportationMax = min((94-25), 80) * min(30, 20) * 0.34 = 469.2
+      // calculatedDataTotalTransportationAllowance = calculatedDataAdditionalTransportationMax
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataAdditionalTransportationMax,
+      ).toBe(469.2);
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(469.2);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm less than 280 " +
```

Comment:
Same for the tests:
https://github.com/bcgov/SIMS/pull/2824/files/58233ccb7c98c92207a61dc9b3c9696f2f38c0a7#r1492915880

Reply:
The E2E test files have been updated. Please have a look.